### PR TITLE
Test plan task

### PR DESCRIPTION
**Pull request by triptijain2112**
**Task name**: <Name of changed or new pipeline task>

**Description**: <Describe your changes here>

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: make-options.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
         "AzureCloudPowerShellDeploymentV2",
         "AzureContainerAppsV0",
         "AzureContainerAppsV1",
+		"AzureDevOpsTestPlanV0",
```
Comment (max-zaytsev) [inline make-options.json:19]: Can you please fix the indentation?

Comment (qianz2) [inline make-options.json:19]: Could you try add the task name to a Node20 section and rebuild it locally?

File: package-lock.json [hunk 2]
```diff
@@ -1,8 +1,3688 @@
 {
   "name": "Agent.Tasks",
   "version": "0.6.0",
-  "lockfileVersion": 1,
```
Comment (max-zaytsev) [inline package-lock.json:4]: I think this should be rolled back. Can you please use node 10 to build tasks locally?

File: package-lock.json [hunk 3]
```diff
@@ -1,8 +1,3688 @@
...
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
```
Comment (qianz2) [inline package-lock.json:7]: Are the changes in this file necessary?

File: make-options.json (hunk 1 above)
Comment (triptijain2112) [reply make-options.json]: Fixed!

Comment (triptijain2112) [reply make-options.json]: Added and built code locally.

File: package-lock.json (hunk 2 above)
Comment (triptijain2112) [reply package-lock.json]: Reverted changes in package-lock.json

File: package-lock.json (hunk 3 above)
Comment (triptijain2112) [reply package-lock.json]: Reverted changes in package-lock.json

**Pull request closed without merging**